tasks/yum.yml

```diff
@@ -21,7 +21,10 @@
     - policycoreutils-python
 
 - name: ensure supervisor is started
-  service: name=supervisord state=started
+  service: name=supervisord state=started enabled=yes
```

Comment:
so by default these two are not started at boot? Anyway adding that doesn't hurt